### DISCUSSION TITLE
For h265 rtsp, HLS is unable to find a video or audio track

### DISCUSSION THREAD

**Discussion by Prakash2101** [bluenviron/mediamtx#468] [Questions]
Which version are you using?
v0.16.4

Which operating system are you using?
Linux (K8s Cluster)

I have deployed simple rtsp server on k8s cluster, h264 codec rtsp is working fine but when I used h265 codec format it's HLS url is not working and it is not able to create .m3u8 file, it is giving the error of " unable to find a video or audio track". Seems like it doesn't have h265 decoder.

![image](https://user-images.githubusercontent.com/61348510/126153481-bc8664f5-6ec2-4a8b-a035-1ff55df81e73.png)

**Comment by aler9** [comment]
HLS at the moment requires an H264 track and/or an AAC track. The error `unable to find a video or audio track` means that the publisher of the track is providing none of that, but is probably providing streams encoded with different codecs.

[a comment by an automated account is not shown]
